### PR TITLE
Mark mac_ios_platform_channels_benchmarks_ios not flaky

### PR DESCRIPTION
**Pull request by jmagman**
`mac_ios_platform_channels_benchmarks_ios` has passed in prod since #82359.

### CODE REVIEW

**Review by CaseyHillers — APPROVED** [review]

**Comment by gaaclarke** [comment]
Don't land this until https://github.com/flutter/flutter/pull/82762 lands.  The test is in fact flakey until that lands.

**Comment by jmagman** [comment]
Huh it doesn't look like `mac_ios_platform_channels_benchmarks_ios` has ever flaked in prod, unlike `platform_channels_benchmarks`.  Beating the `await` on that platform I guess.

Thanks, @gaaclarke will hold off, or I can close this and let you handle it.  I was doing a sweep of `flaky` prod tests that have been passing for awhile, since sometimes we forget to go back and mark them unflaky.

**Comment by jmagman** [comment]
https://github.com/flutter/flutter/pull/82762 has merged.

**Pull request merged**
